packages/cli/src/commands/components/pull/index.ts

```diff
@@ -25,7 +27,14 @@ componentsCommand
 
     // Command options
     const { space, path } = componentsCommand.opts();
-    const { separateFiles, suffix, filename = 'components' } = options;
+    const {
+      separateFiles = false,
+      suffix,
+      filename = 'components',
+    } = options;
+    // `--path` overrides remain command-scoped; fallback keeps the historic .storyblok output.
+    const resolvedBaseDir = path ?? DEFAULT_STORAGE_DIR;
+    const componentsOutputDir = join(resolvedBaseDir, 'components', space);
```

Comment:
suggestion (non-blocking): I think we could use `resolveCommandPath(directories.components, space, path)` here and use the `DEFAULT_STORAGE_DIR` in `resolvePath`.